woocommerce/woocommerce
# Fix Button block markup for Onboarding homepage

### Changes proposed in this Pull Request:

Update the block markup in the Homepage Onboarding creation to utilize the latest markup for nested Button block within a Buttons block, while maintaining the centering of the button.

Closes #33933 .

### How to test the changes in this Pull Request:

1. Tested with both TwentyTwentyTwo theme _and_ the Storefront theme latest.
2. Created a fresh WP install, added Woo trunk with code fix
3. Walked through onboarding to create homepage and verify homepage output on frontend.

![Screen Shot 2022-07-18 at 4 02 35 PM](https://user-images.githubusercontent.com/405912/179610087-81bf0c27-0573-459d-b50b-58293b9406e9.png)

![Screen Shot 2022-07-18 at 4 03 11 PM](https://user-images.githubusercontent.com/405912/179610415-86606058-3504-4c09-906b-ffd7719d9edd.png)

### Other information:

-   [ ] Have you successfully run tests with your changes locally? **n/a**
-   [ ] Have you created a changelog file for each project being changed, ie `pnpm changelog add --filter=<project>`? **No, but I can and will if it is deemed necessary.**

### FOR PR REVIEWER ONLY:
